examples/local/cluster.tmpl.yaml

```diff
@@ -57,14 +57,15 @@ spec:
 
   azure:
     keyVault:
-      name: "$(CLUSTER_NAME)-vault"
+      name: "${CLUSTER_NAME}-vault"
     location: "${AZURE_LOCATION}"
     storage:
-      accountType: "Standard_LRS",
+      accountType: "Standard_LRS"
```

Comment:
This is an error I encountered :)

Reply:
Sorry not sure how that comma got there!

Reply:
@kopizko Do you still get the client ID error? I just repeated the steps and can't repro. Does your config map have this structure?

```
kubectl get cm azure-operator-configmap -o yaml 

apiVersion: v1
data:
  config.yml: |
    service:
      azure:
        clientid: "xxxxx"
        clientsecret: "xxxxx"
        subscriptionid: "xxxxx"
        tenantid: "xxxxx"
        template:
          uri:
            version: "master"
      kubernetes:
        incluster: true
    server:
      listen:
        address: 'http://0.0.0.0:8000'
```